storage/Utils/StorageDefines.h

```diff
@@ -98,17 +98,17 @@
 #define DASDFMTBIN "/sbin/dasdfmt"
 #define DASDVIEWBIN "/sbin/dasdview"
 
-#define UDEVADMBIN "/sbin/udevadm"
-#define UDEVADMBIN_SETTLE UDEVADMBIN " settle --timeout=20"
+#define UDEVADM_BIN "/sbin/udevadm"
+#define UDEVADM_BIN_SETTLE UDEVADM_BIN " settle --timeout=20"
 
 #define RPCBINDBIN     "/sbin/rpcbind"
 
 #define EFIBOOTMGRBIN "/usr/sbin/efibootmgr"
 
-#define NTFSRESIZEBIN "/usr/sbin/ntfsresize"
+#define NTFSRESIZE_BIN "/usr/sbin/ntfsresize"
 #define XFSGROWFSBIN  "/usr/sbin/xfs_growfs"
 #define REISERFSRESIZEBIN "/sbin/resize_reiserfs"
-#define EXT2RESIZEBIN "/sbin/resize2fs"
+#define RESIZE2FS_BIN "/sbin/resize2fs"
```

Comment:
NP: Use `/usr/sbin` instead of just `/sbin` for consistency?

On my (TW) system I have all those commands in `/usr/sbin` and just symlinks to them in `/sbin`.
I don't think that those symlinks will go away anytime soon, but still with all the other commands here using the `/usr/sbin` path it would only be consistent to use it here as well.